ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -828,9 +854,23 @@ bool UncoupledProcessesTimeLoop::loop()
         {
             auto& pcs = spd->process;
             auto const& x = *_process_solutions[pcs_idx];
-            _output->doOutputLastTimestep(pcs, spd->process_output,
-                                          accepted_steps + rejected_steps, t,
-                                          x);
+
+            if (is_staggered_coupling)
```

Comment:
This if-then-else looks almost identical to an above version with dt=0. Could be this deduplicated?

Reply:
Added a generic member function to UncoupledProcessesTimeLoop to handle the ouput, with which all related code duplications is gone. Please review it in the commit entitled with ``[Output] Added a generic output member to UncoupledProcessesTimeLoop``